### DISCUSSION TITLE
Configureable Options

### DISCUSSION THREAD

**Discussion by Pqko0** [Paymenter/Paymenter#227] [Ideas]
When you purchase a product with a extra config, you pay extra but in the invoice or on the customer page it doesn't mention and you can't even edit it after the purchase. So if I wanted to upgrade my ram for example, I would have to repurchase the server and move all of the files.
Basically, make it so it shows on invoice and on the admin panel, and make it to it can be updated/changed later.

**Comment by finnjosten** [comment]
Yeah this would be quite nice to have!
